packages/core/src/services/aws-saml-assertion-extraction-service.ts

```diff
@@ -14,10 +14,14 @@ const authenticationUrlRegexes = new Map([
       /^https:\/\/login\.okta\.com\/.*/,
       /^https:\/\/accounts\.google\.com\/ServiceLogin.*/,
       /^https:\/\/login\.microsoftonline\.com\/.*\/oauth2\/authorize.*/,
+      /^https:\/\/account\.activedirectory\.windowsazure\.com\/applications\/signin\/*?tenantId=*/,
```

Comment:
Adding account.activedirectory.windowsazure.com may not be necessary. If I recall from when I was working through this with the Noovolari team a while back this URL redirects a few times and lands on login.microsoftonline.com, which is where the authentication process begins.

If I recall from our Slack conversation you were getting the authentication window without this but were landing on the AWS Console Page due to the missing govcloud regex that you added below.

Reply:
Thanks! I removed the regex and it works fine.